test/models/batch_ingest_file_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class BatchIngestFileTest < ActiveSupport::TestCase
+
+  def setup
```

Comment:
I *think* this is supposed to be 
```suggestion
  setup do
```

Reply:
Awesome catch 😅. Will update 👍